composer.json

```diff
@@ -418,9 +419,6 @@
                 "Sheets API V3 deprecated": "patches/google_sheets_v3_api_deprecated-3228843.patch"
             }
         },
-        "installer-types": [
-            "library"
-        ],
```

Comment:
Removed because it was causing a problem with oomphinc https://github.com/oomphinc/composer-installers-extender/issues/26